src/gui_common/CollapsibleList.cs

```diff
@@ -120,6 +120,18 @@ public void RemoveItem(string name)
         items.Remove(found);
     }
 
+    public void RemoveAllOfType<T>()
+        where T : Control
+    {
+        var found = items.FindAll(item => item is T);
+
+        foreach (var item in found)
+        {
+            item.DetachAndQueueFree();
```

Comment:
Does this really need this? Or would QueueFree or Free be enough?

Reply:
DetachAndQueueFree is used in RemoveItem so I assume it's preferred

Reply:
It's not really preferred, the order goes: (most preferred) QueueFree, Free, DetachAnd_ (group of methods, these are the least preferred and there's even an issue about removing these entirely).

Reply:
Useful to know, thanks!